src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -211,26 +315,86 @@ unsigned WaveTrackAffordanceControls::CaptureKey(wxKeyEvent& event, ViewInfo& vi
 {
     const auto keyCode = event.GetKeyCode();
     if (!(keyCode == WXK_RETURN || keyCode == WXK_NUMPAD_ENTER || keyCode == WXK_TAB))
+    if (!mTextEditHelper && !(keyCode == WXK_RETURN || keyCode == WXK_NUMPAD_ENTER || keyCode == WXK_TAB))
```

Comment:
I think you meant to replace the line before this new line.  Not to add a new line.  But the code is correct!  Just not simplified